vividus-extension-selenium/src/main/java/org/vividus/selenium/WebDriverProvider.java

```diff
@@ -29,9 +29,9 @@
 
 public class WebDriverProvider implements IWebDriverProvider
 {
-    private IVividusWebDriverFactory vividusWebDriverFactory;
-    private final ConcurrentLinkedQueue<WebDriver> webDrivers = new ConcurrentLinkedQueue<>();
+    @Inject private IVividusWebDriverFactory vividusWebDriverFactory;
```

Comment:
what's the reason of this change?

Reply:
for non-specifiing a concrete implementation